prober/prober.go

```diff
@@ -23,3 +23,9 @@ import (
 )
 
 type ProbeFn func(ctx context.Context, target string, config config.Module, registry *prometheus.Registry, logger log.Logger) bool
+
+const (
+	helpSSLEarliestCertExpiry     = "Returns last SSL chain expiry in unixtime"
+	helpSSLChainExpiryInTimeStamp = "Returns last SSL chain expiry in timestamp"
+	helpProbeTLSInfo              = "Contains the TLS version used"
```

Comment:
nit: I am leaning on the side of using `Returns the TLS version used, or NaN when unknown` for the helpProbeTLSInfo. it's bit more explict. 

feel free to ignore this if you feel otherwise.